test/acceptance/behave/README.md

```diff
@@ -6,11 +6,19 @@ Folder for acceptance tests of context broker NGSI v2.In this framework we are u
 
 ### Prerequisites:
 
-- Python 2.6 or newer
+- Python 2.7.x (One way would be SCL - The Software Collections Repository)
 - pip installed (http://docs.python-guide.org/en/latest/starting/install/linux/)
 - virtualenv installed (pip install virtualenv) (optional).
 Note: We recommend the use of virtualenv, because is an isolated working copy of Python which allows you to work on a specific project without worry of affecting other projects.
 
+
+# Requirements to fabric (http://www.fabfile.org/)
+```
+     yum install gcc python-devel
+```
+   Fabric is a Python (2.5-2.7) library and command-line tool for streamlining the use of SSH for application deployment or systems administration tasks.
```

Comment:
In my opinion, it would be clearer if the yum command should be shown after the line explaining what Fabric is. In addition, it should be explained when this step is needed. I mean, I (as a reader) I'm reading this paragraph but it is unclear if I have to install or not in order to run behave tests.